### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by eliekh05** [Homebrew/discussions#3097] [Tap maintenance and Homebrew development] [answered]
```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Library/Frameworks/Python.framework/Versions/3.10/bin/python3.10-config
  /Library/Frameworks/Python.framework/Versions/3.10/bin/python3-config

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc
Task finished: (1) at 3/19/22, 11:32 PM!
```

**Comment by SMillerDev** [comment]
Help with what?

**Reply by eliekh05** [reply]
Warning: "config" scripts exist outside your system or Homebrew directories.
./configure scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
/Library/Frameworks/Python.framework/Versions/3.10/bin/python3.10-config
/Library/Frameworks/Python.framework/Versions/3.10/bin/python3-config

**Comment by carlocab** [comment, marked as answer]
> ```
> Please note that these warnings are just used to help the Homebrew maintainers
> with debugging if you file an issue. If everything you use Homebrew for is
> working fine: please don't worry or file an issue; just ignore this. Thanks!
> ```

Is anything you use Homebrew for not working fine?

**Reply by eliekh05** [reply]
ok thank you
